ga4gh/backends.py

```diff
@@ -186,19 +186,19 @@ def searchVariants(self, request):
         else:
             readCols = self._table.columns()[:self._firstSamplePosition]
             callSetIds = request.callSetIds
-            for csid in request.callSetIds:
-                cols = [c for name, c in self._sampleCols[csid]]
+            for callSetId in request.callSetIds:
+                cols = [col for name, col in self._sampleCols[callSetId]]
                 readCols.extend(cols)
         # Now we get the row positions for the sample columns
         sampleRowPositions = {}
-        j = self._firstSamplePosition
-        for csid in callSetIds:
-            l = []
-            for c in self._sampleCols[csid]:
-                l.append(j)
-                j += 1
-            sampleRowPositions[csid] = l
-        v = []
+        currentRowPosition = self._firstSamplePosition
+        for callSetId in callSetIds:
+            sampleRowPositionsList = []
+            for col in self._sampleCols[callSetId]:
```

Comment:
https://docs.python.org/2/library/itertools.html#itertools.count I believe is the best way to do that.